tutorials/k8s-fluentbit-observability/index.mdx

```diff
@@ -0,0 +1,233 @@
+
+First, generate a Cockpit token:
+1. Go to your Cockpit in the **API and Tokens tab**.
+2. Navigate to the **Token section** and click on **Generate a token**.
+3. Choose both Push permissions for metrics and logs.
+
+<Lightbox src="scaleway-cockpit-token-permissions.webp" alt="" />
+
+In the `values.yaml` file, create a section named `config.outputs`:
+
+```yaml
+  outputs: |
+    [OUTPUT]
+        Name                  prometheus_remote_write
+        Match                 node_metrics
+        Host                  <...>
+        Port                  443
+        Uri                   /api/v1/push
+        Header                Authorization Bearer <...>
+        Log_response_payload  false
+        Tls                   on
+        Tls.verify            on
+        Add_label             job kapsule-metrics
+    [OUTPUT]
+        Match                 logs.*
+        Name                  loki
+        Host                  <...>
+        Port                  443
+        Tls                   on
+        Tls.verify            on
+        Label_map_path        /fluent-bit/etc/labelmap.json
+        Auto_kubernetes_labels on
+        Http_user             nologin
+        Http_passwd           <...>
+```
+
+Fill the blanks as follows:
+1. `Host` from the first subsection: paste your Metrics API URL defined in the **API and Tokens tab** section from the Cockpit. Remove the `https://` protocol.
+2. `Header`: Next to `Bearer`, paste the token generated in the previous step.
+3. `Host` from the second subsection: paste your Logs API URL defined in the **API and Tokens tab** section from the Cockpit. Remove the `https://` protocol.
+4. `Http_passwd`: paste the token generated in the previous step.
+
+In the first subsection, the `prometheus_remote_write` plugin is used to send metrics to the [Prometheus](https://prometheus.io/) server of your Cockpit using the remote write protocol.
+In the second subsection, the `loki` plugin is used to send logs to the [Loki](https://grafana.com/oss/loki/) server of your Cockpit, using the field mapping from `labelmap.json` defined above.
+
+## Installing Fluent Bit
+
+In the same directory as your `values.yaml` file, run:
+
+```
+helm upgrade --install fluent-bit fluent/fluent-bit -f .\values.yaml
+```
+
+Fluent Bit has been installed in your Kapsule cluster. You should see a `DeamonSet` named `fluent-bit` with running pods on all of your nodes.
+
+## Visualizing Kapsule logs and metrics
+
+Logs and metrics from your Kubernetes cluster can now be found in your Cockpit's dashboard
+
+### Exploring metrics
+
+Grafana has a built-in dashboard for visualizing node metrics.
+1. Go to **Dashboards** in your Grafana instance
+2. Click **New**, **Folder** and name it `Kapsule`
+3. Click **New**, **Import** and paste the following URL in the **Import via grafana.com** field:
+    ```
+    https://grafana.com/grafana/dashboards/405-node-exporter-server-metrics/
```

Comment:
I think this dashboard `https://grafana.com/grafana/dashboards/1860-node-exporter-full/` is better, what do you think ? The one you provided is from 2018 and this one is up to date. (And nicer but it's a matter of taste)